libs/viewer/src/Settings.cpp

```diff
@@ -1168,7 +1170,8 @@ static std::ostream& operator<<(std::ostream& out, const LightManager::ShadowOpt
     math::float3 splitsVector = { splits[0], splits[1], splits[2] };
     return out << "{\n"
         << "\"vsm\": {\n"
-        << "\"msaaSamples\": " << int(in.vsm.msaaSamples) << "\n"
+        << "\"msaaSamples\": " << int(in.vsm.msaaSamples) << ",\n"
+        << "\"blurStandardDeviation\": " << int(in.vsm.blurStandardDeviation) << "\n"
```

Comment:
I think the cast-to-int is unintentional here.

Reply:
Thanks!